source/messaging/keyboard-shortcuts.rst

```diff
@@ -116,9 +116,12 @@ The following keyboard shortcuts are supported in all supported browsers and in
 +----------------------------+---------------------------+----------------------------------------------------------------------------------+
 | CTRL+.                     | CMD+.                     | Open and close the right-hand sidebar.                                           |
 +----------------------------+---------------------------+----------------------------------------------------------------------------------+
+| CTRL+SHIFT+.               | CMD+SHIFT+.               | Expand or reduce the width of the open right-hand sidebar.                       |
```

Comment:
Wondering if "Expand or shrink" may work better as they're more similar in function than Expand or reduce?